apps/astarte_appengine_api/lib/astarte_appengine_api/config.ex

```diff
@@ -140,16 +140,16 @@ defmodule Astarte.AppEngine.API.Config do
           :astarte_data_updater_plant,
           :clustering_strategy,
           os_env: "CLUSTERING_STRATEGY",
-          type: Astarte.AppEngine.API.ClusteringStrategy,
+          type: Astarte.AppEngine.API.Config.ClusteringStrategy,
           default: "none"
 
-  @envdoc "The Kubernetes selector to use when `kubernetes` Erlang clustering strategy is used. Defaults to `app=astarte-data-updater-plant`."
-  app_env :clustering_kubernetes_selector,
+  @envdoc "The Endpoint label to use to query Kubernetes to find data updater plant instances. Defaults to `app=astarte-data-updater-plant`."
+  app_env :dup_clustering_kubernetes_selector,
           :astarte_data_updater_plant,
-          :clustering_kubernetes_selector,
-          os_env: "CLUSTERING_KUBERNETES_SELECTOR",
+          :dup_clustering_kubernetes_selector,
+          os_env: "DUP_CLUSTERING_KUBERNETES_SELECTOR",
```

Comment:
Nit: you are looking for a DUP cluster and we're using full names in env vars, so I'd suggest
```suggestion
          os_env: "DATA_UPDATER_PLANT_CLUSTER_KUBERNETES_SELECTOR",
```